interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: An instance variable is a variable used to pass infortmation through classes in ruby that begins with @ 
 
-  Researched answer:
+  Researched answer: An instance variable is a variable that is accessible in any instance method in a particular instance of a class
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: they are ruby methods that begin with do and end 
 
-  Researched answer:
+  Researched answer:Ruby blocks are anonymous functions that can be passed into methods.
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: it means that it has a very specific return it doesnt hoist the way that java does. 
 
-  Researched answer:
+  Researched answer:Any statement in Ruby returns the value of the last evaluated expression
 
 
 
 4. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: object oriented programming is like ruby where everything is an object versus java-script where is more functional functional programming 
```

Comment:
Okay you've started a very helpful explanation. JS is also naturally OOP but can be utilized in an FP way. Write another sentence or two when you're explaining a topic like this to help clarify your thoughts.